tests/e2e/specs/stories-editor/page.js

```diff
@@ -218,4 +248,102 @@ describe( 'Story Page', () => {
+		const storySettingsAdvanceAfterSelector = '.components-panel__body.amp-story-settings-panel .components-select-control__input:nth-of-type(1)';
+		const storySettingsAdvanceAfterDurationSelector = '.components-panel__body.amp-story-settings-panel [aria-label="Time in seconds"]';
+
+		await page.click( documentPanelSelector );
+		await page.waitForSelector( storySettingsPanelSelector );
+
+		await page.click( storySettingsPanelSelector );
+		await page.waitForSelector( storySettingsAdvanceAfterSelector );
+
+		expect( await page.evaluate( ( selector ) => {
+			return document.querySelector( selector ).value;
+		}, storySettingsAdvanceAfterSelector ) ).toBe( 'time' );
+
+		expect( await page.evaluate( ( selector ) => {
+			return document.querySelector( selector ).value;
+		}, storySettingsAdvanceAfterDurationSelector ) ).toBe( '2' );
+	} );
+
+	it( 'should save the story settings correctly', async () => {
+		const documentPanelSelector = 'button.edit-post-sidebar__panel-tab[data-label="Document"]';
+		const storySettingsPanelSelector = '.amp-story-settings-panel';
+		const storySettingsAdvanceAfterSelector = '.amp-story-settings-panel .components-select-control__input:nth-of-type(1)';
+
+		await page.click( documentPanelSelector );
+		await page.waitForSelector( storySettingsPanelSelector );
+
+		await page.click( storySettingsPanelSelector );
+		await page.waitForSelector( storySettingsAdvanceAfterSelector );
+
+		await page.select( storySettingsAdvanceAfterSelector, 'auto' );
+
+		await saveDraft();
+		await page.reload();
+
+		await page.click( documentPanelSelector );
+		await page.waitForSelector( storySettingsPanelSelector );
+
+		await page.click( storySettingsPanelSelector );
+		await page.waitForSelector( storySettingsAdvanceAfterSelector );
+
+		expect( await page.evaluate( ( selector ) => {
+			return document.querySelector( selector ).value;
+		}, storySettingsAdvanceAfterSelector ) ).toBe( 'auto' );
+	} );
+
+	it( 'should apply story settings to newly created pages', async () => {
+		await page.click( '.block-editor-inserter .editor-inserter__amp-inserter' );
+		await page.waitForXPath( `//div[contains(@class, 'amp-story-page-number') and contains(text(), 'Page 2')]` );
+
+		const pageAdvancementSelector = '.components-select-control__input';
+		await page.waitForSelector( pageAdvancementSelector );
+
+		const secondsSelector = 'input[aria-label="Time in seconds"]';
+		await page.waitForSelector( secondsSelector );
+
+		expect( await page.evaluate( ( selector ) => {
+			return document.querySelector( selector ).value;
+		}, pageAdvancementSelector ) ).toBe( 'time' );
+
+		expect( await page.evaluate( ( selector ) => {
+			return document.querySelector( selector ).value;
+		}, secondsSelector ) ).toBe( '2' );
+	} );
+
+	it( 'should not affect story settings when individual page settings are changed', async () => {
+		await page.click( '.block-editor-inserter .editor-inserter__amp-inserter' );
```

Comment:
Why not just use await insertBlock( 'Page' );?

Reply:
@spacedmonkey Because for some reason I completely ignored its existence. :)